src/hindi.js

```diff
@@ -45,5 +45,4 @@ class Hindi {
   }
 }
 
-export { Hindi };
-
+export default { Hindi };
```

Comment:
Same as for French,
default won't work here. Hindi is a class, not a constructor.